src/test/scala/org/broadinstitute/hail/methods/ImputeSexSuite.scala

```diff
@@ -64,33 +63,32 @@ class ImputeSexSuite extends SparkSuite {
           val (_, imputedSexQuery) = s.vds.querySA("if (sa.imputesex.isFemale) 2 else 1")
           val (_, fQuery) = s.vds.querySA("sa.imputesex.Fstat")
 
-          val hailResult = sc.parallelize(s.vds.sampleIdsAndAnnotations.map { case (sample, sa) =>
-            (sample, (imputedSexQuery(sa), fQuery(sa)))
-          })
+          val hailResult = s.vds.sampleIdsAndAnnotations.map { case (sample, sa) =>
+            (sample, (imputedSexQuery(sa).map(_.asInstanceOf[Int]), fQuery(sa).map(_.asInstanceOf[Double] + .0005)))
```

Comment:
I can't quite see why increasing the MAF above and adding 0.0005 correspond to what plink is doing.